src/angular/styles/_typography.scss

```diff
@@ -2627,7 +2638,7 @@ $panelBorderRadius: 2;
     margin: sbb.pxToRem(10) 0;
     padding: 0;
     border: none;
-    border-top: var(--sbb-border-width-thin) solid var(--sbb-color-cloud);
+    border-top: var(--sbb-border-width-thin) solid var(--sbb-usermenu-divider-color);
```

Comment:
Since we now use it for the panel globally, I suggest adding another variable section clalled `// Panel` and putting the variable `--sbb-panel-divider-color` there. z--sbb-usermenu-divider-color` can be removed.